cfme/tests/configure/test_access_control.py

```diff
@@ -2101,11 +2102,91 @@ def test_tenant_visibility_miq_ae_namespaces_all_parents():
     pass
 
 
-@pytest.mark.manual
+def create_user(appliance):
+    """This function will create user, group, and role with all product feature
+    by clicking individually"""
+    COMMON_FEATURES = [
+        "Services",
+        "Compute",
+        "Configuration",
+        "Networks",
+        "Storage",
+        "Control",
+        "Automation",
+        "Monitor",
+        "Help",
+        "API",
+        "Service UI",
+    ]
+
+    FEATURES_511 = COMMON_FEATURES + [
+        "Overview",
+        "Migration",
+        "User Settings",
+        "All VM and Instance Access Rules",
+        "Main Configuration",
+    ]
+
+    FEATURES_510 = COMMON_FEATURES + [
+        "Cloud Intel",
+        "Optimize",
+        "Access Rules for all Virtual Machines",
+    ]
+
+    features = FEATURES_511 if appliance.version > "5.11" else FEATURES_510
+    product_features = [[["Everything", feature], True] for feature in features]
+    role = appliance.collections.roles.create(
+        name=fauxfactory.gen_alpha(15, start="API-role-"),
+        product_features=[[["Everything"], False]] + product_features,
+    )
+    # creating group
+    group = appliance.collections.groups.create(
+        description=fauxfactory.gen_alphanumeric(22, "group_description_"),
+        role=role.name,
+    )
+
+    creds = Credential(
+        principal=fauxfactory.gen_alphanumeric(start="user_"),
+        secret=fauxfactory.gen_alphanumeric(),
+    )
+    # user with above group
+    user = appliance.collections.users.create(
+        name=fauxfactory.gen_alphanumeric(start="user_"),
+        credential=creds,
+        email=fauxfactory.gen_email(),
+        groups=group,
+    )
+    return role, group, user
+
+
+@pytest.fixture
+def testing_vm(appliance):
```

Comment:
@dgaikwad this should use `create_vm`https://github.com/ManageIQ/integration_tests/blob/e5755e895ea04688f650d40222b11d994f935196/cfme/fixtures/vm.py#L130

With the `provider` passed in on the test case.